Formula/plank.rb

```diff
@@ -0,0 +1,33 @@
+class Plank < Formula
+  desc "Framework for generating immutable model objects"
+  homepage "https://pinterest.github.io/plank/"
+  url "https://github.com/pinterest/plank/archive/v1.2.tar.gz"
+  sha256 "9ee8c248d7b4488f7f952b01d913b34cd83bd40111092fd49015d1c63b9e885d"
+  head "https://github.com/pinterest/plank.git"
+
+  depends_on :xcode => ["9.0", :build]
+
+  def install
+    system "make", "archive"
+    bin.install ".build/release/plank"
```

Comment:
please request a `make install` target upstream

Reply:
Can you explain this a bit more? Using `bin.install` seemed like the correct way to install binaries from the documentation

Reply:
We don't want to have to keep track of what should or shouldn't be installed and where if it later changes. So `make install PREFIX=/usr/local/Cellar/plank/1.2` should be possible.

Reply:
Thanks for explaining, it is addressed now